app/views/expeditions/index.html.slim

```diff
@@ -8,9 +8,9 @@ ul
     .col-sm-6
       p
         a href=expedition_path(expd)
-          = "#{expd.expedition_type.name} | #{expd.dragon.name}"
+          = "\"#{expd.expedition_type.name}\" - #{expd.dragon.name} (ends at: #{expd.start_time + expd.expedition_type.longevity})"
       p
-        = link_to 'Bring back', expedition_path(expd),
+        = link_to_if expd.start_time + expd.expedition_type.longevity < DateTime.now, 'Bring back', expedition_path(expd),
```

Comment:
można utworzyć metodę w modelu, ponieważ to się powtarza często
`class Expedition < ....

def end_time
 start_time + expedition_type.longevity
end

def finished?
  end_time < DateTime.current # uwzględnia strefę czasową, rubocop też może krzyczeć że .now jest złe
end
```